leechcore_device_microvmi/leechcore_device_microvmi.c

```diff
@@ -24,48 +24,84 @@ static VOID DeviceMicrovmi_Close(_Inout_ PLC_CONTEXT ctxLC)
   microvmi_destroy(driver);
 }
 
+static bool parse_url_args(PLC_CONTEXT ctxLC, DriverInitParamsFFI* init_params)
+{
+    char* url_device = ctxLC->Config.szDevice;
```

Comment:
good catch.
the second one however should remain non const, since it will be used as input for `strtok_r`, which modifies the string (inserts `\0`).
hence the `strdup`